cpu/cc2538/dev/gpio.h

```diff
@@ -318,7 +318,7 @@ typedef void (* gpio_callback_t)(uint8_t port, uint8_t pin);
  * \return A pin mask which can be used as the PIN_MASK argument of the macros
  * in this category
  */
-#define GPIO_PIN_MASK(PIN) (1 << (PIN))
+#define GPIO_DIO_ALL_MASK(PIN) (1 << (PIN))
```

Comment:
I have renamed it because ti changed it on the Ti's cc26xxware, you can check it on [Release notes v2.23.00.16374](http://processors.wiki.ti.com/index.php/CC26xxWare) or in the new code directly [new cc26xxware](https://github.com/cquijano/cc26xxware/tree/new_cc26xxware), maybe I should add another define on ti-lib.h like:
```
#define GPIO_PIN_MASK(...) GPIO_DIO_ALL_MASK(...)
```